module/VuFindSearch/src/VuFindSearch/Backend/ProQuestFSG/Connector.php

```diff
@@ -105,6 +105,11 @@ public function search(ParamBag $params, $offset, $limit)
         $params->set('operation', 'searchRetrieve');
         $params->set('recordSchema', 'marcxml');
 
+        // TEMP -- blender wants to start at zero
```

Comment:
Yeah... haven't solved this yet.  ProQuest's offset starts at 1.

Reply:
I think it would be better for offset to always be 0-based in VuFind, because it's displacement from the first record, i.e. for offset it doesn't really matter if first record is 0 or 1 in the API, because it's always relative to the first record. Adjust as necessary in the Proquest API calls instead.

Reply:
I made this change -- restoring the offset to be 0-based, and doing the ProQuest increment within its Connector class.